docs/src/domains/steps/subdomains.md

```diff
@@ -65,9 +67,7 @@ that project now owns `example.com` as far as Platform.sh is concerned
 and no other project can have a domain anywhere in `*.example.com`.
 (Multiple subdomains within that same project are perfectly fine.)
 
-In most cases, that's a desirable added layer of security.
-But you may run into a problem when you want multiple subdomains from the same organization as separate projects.
-(Multiple departments at the same university, for instance.)
+The subdomain hijacking protection can cause issues if you want multiple subdomains from the same organization as separate projects such as for example multiple departments at the same university.
```

Comment:
:hammer_and_wrench: The new version seems longer and more complex without adding much clarity. Unless you have specific concerns about the original that I don't see, the most I could see would be getting rid of the parentheses:
```suggestion
But you may run into a problem when you want multiple subdomains from the same organization as separate projects.
For example, multiple departments at the same university.
```